frontend/src/tabs/account/Account.js

```diff
@@ -342,6 +342,10 @@ const Account = (props) => {
             </div>
           )}
 
+          <p className='Disconnect__button' onClick={() => disconnect()}>
```

Comment:
To keep things consistent, I think using the same styling / classes as the login buttons would be preferred